pkg/controller/status/status.go

```diff
@@ -761,6 +773,9 @@ func (m *statusManager) degradedReason() string {
 	if m.explicitDegradedReason != "" {
 		reasons = append(reasons, m.explicitDegradedReason)
 	}
+	if m.windowsUpgradeDegradedReason != "" {
```

Comment:
do we also need to add a degradedMessage when windowsUpgradeDegradedReason is not empty?